File: CS/WebSite/App_Code/DocumentServerDemoUtils.cs
using System.IO;
using DevExpress.Spreadsheet;
using DevExpress.XtraSpreadsheet.Export;

public class HtmlContentGenerator
{
    Stream stream;
    public HtmlContentGenerator(Stream stream)
    {
        this.stream = stream;
    }
    public void Generate(IWorkbook workbook, int sheetIndex)
    {
        using (MemoryStream ms = new MemoryStream())
        {
            HtmlDocumentExporterOptions options = new HtmlDocumentExporterOptions
            {
                SheetIndex = sheetIndex,
                EmbedImages = true,
                AnchorImagesToPage = true
            };
            workbook.ExportToHtml(ms, options);
            ms.Seek(0, SeekOrigin.Begin);
            ms.CopyTo(this.stream);
        }
    }
    public void Generate(IWorkbook workbook)
    {
        Generate(workbook, 0);
    }
}

File: VB/WebSite/App_Code/DocumentServerDemoUtils.vb
Imports System.IO
Imports DevExpress.Spreadsheet
Imports DevExpress.XtraSpreadsheet.Export

Public Class HtmlContentGenerator
    Private stream As Stream
    Public Sub New(ByVal stream As Stream)
        Me.stream = stream
    End Sub
    Public Sub Generate(ByVal workbook As IWorkbook, ByVal sheetIndex As Integer)
        Using ms As New MemoryStream()
            Dim options As HtmlDocumentExporterOptions = New HtmlDocumentExporterOptions With { _
                .SheetIndex = sheetIndex, _
                .EmbedImages = True, _
                .AnchorImagesToPage = True _
            }
            workbook.ExportToHtml(ms, options)
            ms.Seek(0, SeekOrigin.Begin)
            ms.CopyTo(Me.stream)
        End Using
    End Sub
    Public Sub Generate(ByVal workbook As IWorkbook)
        Generate(workbook, 0)
    End Sub
End Class